### ISSUE TITLE
Full text search

### ISSUE THREAD

**Issue by Januzz** [jmjeong/alfred-extension#1]
Thanks for the great workflow!

If I understand correctly, pba query should do full text search if one has an archive pinboard account, right? 

For me, it seems to search the titles only.

**Comment by jmjeong** [comment]
pda searches title, description, and tags only. It doesn't search archived pinboard content.

**Comment by Januzz** [comment]
Would you consider implementing it as a feature?

**Comment by jmjeong** [comment]
To implement it, the archived content should be downloaded to mac for the fast search. But there is no pinboard api to download its content.

**Comment by Januzz** [comment]
That's true. And I assume you wouldn't like to implement it as a (slow) web based search?

**Comment by jmjeong** [comment]
You can try 'Web search' feature in Alfred App.

Because I don't have the archived account, I can't test it. But I think it
will work.


2015-01-07 10:56 GMT+09:00 Januzz <notifications@github.com>:

> That's true. And I assume you wouldn't like to implement it as a (slow)
> web based search?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmjeong/alfred-extension/issues/1#issuecomment-68968012>
> .
>



-- 
Jaemok Jeong

**Comment by Januzz** [comment]
Thanks, I already do. But this takes me to the website. Would be great to display the results in a list in Alfred...

**Comment by jmjeong** [comment]
I am sorry but it is hard to support this feature without API support.
​How about suggesting
<support@pinboard.in>
​ to add such API?

2015-01-07 11:03 GMT+09:00 Januzz <notifications@github.com>:

> Thanks, I already do. But this takes me to the website. Would be great to
> display the results in a list in Alfred...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmjeong/alfred-extension/issues/1#issuecomment-68968582>
> .
>



-- 
Jaemok Jeong

**Comment by Januzz** [comment]
Will do, thanks!

Am 06.01.2015 um 18:08 schrieb Jaemok Jeong <notifications@github.com>:

**Comment by Januzz** [comment]
He says that he plans to include full text search in the next API version. Let's wait and see then :)
